changelog.md

```diff
@@ -1,5 +1,6 @@
 ### v4.5.6
 - Fix: hide edit pencils on iOS on H3, H4, H5 and H6
+- Fix: add a few css properties to read more part of the footer
```

Comment:
I recommend having a placeholder heading for not yet released changes, similar to what [domino](https://github.com/fgnass/domino/commit/e43984f7c80023943b6585652be72aeafb4e2971) does.